<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Yehia-pc" failures="1" tests="1" name="Check Register with Valid Data" time="45.681" errors="0" timestamp="2023-10-10T21:12:25 EET">
  <testcase classname="TestCases.TC01_Registration" name="registerNewUser_P" time="12.165">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.TC01_Registration.registerNewUser_P(TC01_Registration.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- registerNewUser_P -->
</testsuite> <!-- Check Register with Valid Data -->
